src/dapp/utils/api_request_helpers.ts

```diff
@@ -0,0 +1,112 @@
+import { error } from 'console'
+
+//Interfaces
+
```

Comment:
```suggestion
const user_email = process.env.USER_NAME;
if (!user_email) throw new Error("USER_NAME not set")
```
That is a simple version, how you can test, if the variable is set in the moment of starting.

Reply:
will test that